rest/model/admin.go

```diff
@@ -419,6 +427,13 @@ func (a *APIAuthConfig) ToService() (interface{}, error) {
 	if i != nil {
 		ldap = i.(*evergreen.LDAPConfig)
 	}
+	i, err = a.Okta.ToService()
+	if err != nil {
+		return nil, err
+	}
+	if i != nil {
+		okta = i.(*evergreen.OktaConfig)
```

Comment:
`var ok bool`
`okta, ok = i.(*evergreenOktaConfig)`
`if !ok {`

Reply:
Done. I changed this for all the auth configs.